app/api/news/news.controller.js

```diff
@@ -0,0 +1,62 @@
+const Sequelize = require("sequelize");
+const config = require("../../../config");
+const models = require("../../models");
+const utils = require("../../utils/express");
+const mediaUtils = require("../../utils/media");
+
+const Op = Sequelize.Op;
+
+exports.getList = utils.route(async (req, res) => {
+  const lang = req.query.lang || (req.getLocale() || "en");
+
+  const offset = req.query.offset ? parseInt(req.query.offset) : 0;
+  const limit = req.query.limit ? parseInt(req.query.limit) : 10;
+
+  const news = await models.news.findAndCountAll({
+    limit,
+    offset,
+    order: [["created_at", "DESC"]],
```

Comment:
in case we add a `published_at` we would need to check now > published_at and order by publication date